app/disply.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+
+from app.model import Book
+
+
+class Display(ABC):
+    def __init__(self, book: Book) -> None:
+        self.book = book
+
+    @abstractmethod
+    def display(self) -> None:
+        ...
+
+
+class ConsoleDisplay(Display):
+    def display(self) -> None:
+        print(self.book.content)
+
+
+class ReverseDisplay(Display):
+    def display(self) -> None:
+        print(self.book.content[::-1])
```

Comment:
Mistake in the file name + fix imports.
1 blank line before and after class methods/subclasses ;)